schema/reportdb/common/tables/Channel.sql

```diff
@@ -0,0 +1,29 @@
+--
+-- Copyright (c) 2022 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+
+CREATE TABLE Channel
+(
+    mgm_id                    NUMERIC NOT NULL,
+    channel_id                NUMERIC NOT NULL,
+    name                      VARCHAR(256),
+    label                     VARCHAR(128),
+    type                      VARCHAR(50),
+    product_name              VARCHAR(128),
```

Comment:
I think we can remove "product_name" here. The values we set are insufficient and normally a channel belong to multiple products